microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,190 @@
+
+  * CoreWebView2
+    - `BrowserProcessId` property
+    - `ProcessFailed` event
+  * CoreWebView2Environment
+    - `BrowserProcessExited` event
+    - `ProcessInfosChanged` event
+    - `GetProcessInfos` method
+
+  > [!NOTE]
+  > This is an overview at the conceptual level. For detailed API names and related types in your app's framework, see [Process Management](overview-features-apis.md#process-management).
+
+In this document, we discuss how to handle process exits and failures, using the `CoreWebView2.ProcessFailed` and `CoreWebView2Environment.BrowserProcessExited` events.
+
+## Process Events
+The WebView2 Runtime is composed of different types of processes, including:
+  * Main browser process
+  * Renderer processes
+  * Utility processes
+  * GPU process
+
+  > [!NOTE]
+  > This is an illustrative and incomplete list of process kinds. The purpose and management details for these processes is part of the Chromium architecture and beyond the scope of this document, or the WebView2 architecture.
+
+When you create and initialize a WebView2 control, WebView2 will ensure there's a WebView2 Runtime to power your control and connect to its [WebView2 Process Group](process-model.md#processes-in-the-webview2-runtime). Once this connection is established, your control will start monitoring these processes for the following events:
+
+  * **Main browser process exits.** If the main browser process exits for _any reason_, the `CoreWebView2Environment` will raise the `BrowserProcessExited` event. Use this event to synchronize operations involving the WebView2 Runtime resources and lifetime, such as _User Data Folder_ management and updates. See [Main Browser Process Exits](<link>) below for more details.
+  * **Any process failure.** When _any of the processes_ in the WebView2 Runtime fails, the CoreWebView2 will raise the `ProcessFailed` event. Use this event for diagnostics and recovery from failures in the WebView2 processes. See [Process Failures](<link>) below for more details.
+
+  > [!NOTE]
+  > There is some overlap between these two events. For example, a main browser process crash will produce both a `ProcessFailed` event and a `BrowserProcessExited` event, since the main browser process _exited_ because of a failure. See [Main Browser Process Crashes](<link>) for more information about this case.
+
+`CoreWebView2` and `CoreWebView2Environment` report these events so your application can react accordingly. There are multiple scenarios your application can handle through these events.
+
+## Main Browser Process Exits
+The `BrowserProcessExited` event ([COM](<link>), [.NET](<link>), [WinRT](<link>)) indicates that the main browser process has exited and its resources (including its child processes) have been released. This can happen for the following reasons:
```

Comment:
```suggestion
The `BrowserProcessExited` event indicates that the main browser process has exited and its resources (including its child processes) have been released. This can happen for the following reasons:
```
moved link to tab-set at end of section